jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/Main.java

```diff
@@ -70,9 +70,15 @@ public static Map<String, Object> execute(Map<String, String> params) throws Nam
             break;
         }
         ArrayList errors= (ArrayList) errorInfo.get("errors");
+
         if(!errors.isEmpty()){
             //Below logger message is used by datadog to create notification in slack
             log.error("Error occurred in atleast one collector for jobId : {}",jobHint);
+            errors.stream().forEach(error -> {
+                Map<String, Object> err = (Map<String, Object>) error;
+                log.error("error in main - {}", err.get("error"));
```

Comment:
Here for each error you are creating two log statements. Let's just do one log.error with the full error exception for each error?

Reply:
done.